README.md

```diff
@@ -44,6 +44,82 @@ docker build -t pca-assessment .
 We welcome contributions!  Please see [here](CONTRIBUTING.md) for
 details.
 
+## Assessment JSON Field Dictionary ##
+
+The following items are included in the assessment JSON as produced by the `pca-wizard`.
+An example assessment JSON can be found [here](src/assessment/sample_assessment.json).
+
+| Name | Description | Type | Default | Required |
+|------|-------------|:----:|:-------:|:--------:|
+| id | Assessment identifier (e.g. "RV0000"). | string | | yes |
+| timezone | Timezone name based on [pytz](http://pytz.sourceforge.net/) timezones. (e.g. "US/Eastern") | string | | yes |
+| domain | Assessment domain for GoPhish public interface. | string | | yes |
+| target_domain | Approved target domains where all email recipients must reside. | list(string) | | yes |
+| start_date | Assessment start date in 24-hr ISO format with offset. | string | | yes |
+| end_date | Assessment end date in 24-hr ISO format with offset. | string | | yes |
```

Comment:
Please also add examples for all date, domain, URL, and list fields in the README.

Reply:
@dav3r, what are your thoughts on the examples, becoming a new column after the description?  This would allow each line to have an example in a specific location. This column could also hold the links to the example tables for items such as `targets`, `campaigns`, etc.

I am also thinking that there would be value-added in a comment under the headers for `Template Dictionary` and `Group Dictionary` that indicates these are imported from templated files that can be generated using the `pca-wizard-templates` command.

Reply:
In our other projects, we haven't used a separate column for examples.  If you want to do it, I'm game for giving it a shot and seeing how we like it.

I agree that adding comments about `pca-wizard-templates` is an excellent idea. 👍

Reply:
I took a look at the new column idea and see why other projects have not gone this route.  By doing this, the columns became much smaller and did not look good. Going back to the original course of action.

Reply:
Added examples in commit 2c35254 and comment about `pca-wizard-templates` in 457e323.

Reply:
In 4394174, I changed the wording of the comment about the templates in an attempt to make it more accurate.